modules/carousel/jetpack-carousel.js

```diff
@@ -1552,7 +1552,7 @@ jQuery( document ).ready( function( $ ) {
 					// attachment id might no longer match the current attachment id by the time we get the data back or a now
 					// registered infiniscroll event kicks in, so we don't ever display comments for the wrong image by mistake.
 					var current = $( '.jp-carousel div.selected' );
-					if ( current && current.data && current.data( 'attachment-id' ) != args.attachment_id ) {
+					if ( current && current.data && current.data( 'attachment-id' ) !== args.attachment_id ) {
```

Comment:
I was concerned about this change but I've tested and confirmed that we're getting numbers on both sides 👍

Reply:
Thanks for confirming and checking that!

Reply:
This was ignored before, but the ignore line comment was dropped down by prettier.

I think this is the riskiest linter-initiated change out of the bunch and I thank after rebasing on the other fixes I'd prefer to put the ignore comment back on this line.